lib/mongoProvider.js

```diff
@@ -103,7 +103,7 @@ MongoCollection.prototype.find = function (query, req) {
     return self.db().then(function (db) {
         return self.beforeFindListeners.fire(query, req).then(function () {
 
-            return q.ninvoke(db.collection(self.name).find(query), "toArray").then(function(res) {
+            return q.ninvoke(db.collection(self.name).find(query, req), "toArray").then(function(res) {
```

Comment:
Sure...
I have several jsReport servers and I'm working on sharing templates between different dbs. In web console user will select src db, dst db and template name and press share button. That why I need list of templates names.